src/main/java/frc/utils/pathplannerutils/AutonomousChooser.java

```diff
@@ -0,0 +1,22 @@
+package frc.utils.pathplannerutils;
+
+import com.pathplanner.lib.auto.AutoBuilder;
+import edu.wpi.first.wpilibj2.command.Command;
+import org.littletonrobotics.junction.networktables.LoggedDashboardChooser;
+
+public class AutonomousChooser {
+
+    private final LoggedDashboardChooser<Command> chooser;
+
+    public AutonomousChooser(String name, String defaultOption) {
+        this.chooser = new LoggedDashboardChooser<Command>(name, AutoBuilder.buildAutoChooser(defaultOption));
```

Comment:
This will put the chooser in smart dashboard. Maybe we should put it in a special tab (to your choice).